applications/asset_tracker/src/env_sensors/env_sensors.c

```diff
@@ -54,6 +56,20 @@ static struct env_sensor *env_sensors[] = {
 };
 
 static struct k_delayed_work env_sensors_poller;
+static env_sensors_data_ready_cb data_ready_cb;
+static s32_t data_send_interval_s = CONFIG_ENVIRONMENT_DATA_SEND_INTERVAL;
+static bool backoff_enabled;
+static bool initialized;
+
+static inline int submit_poll_work(const s32_t delay_s)
+{
+	return k_delayed_work_submit(&env_sensors_poller, K_SECONDS(delay_s));
+}
+
+int env_sensors_poll(void)
+{
+	return initialized ? submit_poll_work(K_NO_WAIT) : -ENXIO;
```

Comment:
`initialized` variable was already checked in `env_sensors_set_send_interval` function, why do we check it again?

Reply:
`env_sensors_poll` is not static, it is also used in `main.c`

Reply:
Right, I've missed that, thanks.